README.md

```diff
@@ -1,24 +1,911 @@
+  </a>
 
-* Database creation
+<h3 align="center">Tavern Keeper</h3>
 
-* Database initialization
+  <p align="left">
+    This application is a Dungeons and Dragons Encounter Builder, with the intended user being a Dungeon Master. It allows the user to fill out a form to add encounter details including encounter name, party size, party level, summary, description, treasure & rewards, and monsters. Monsters from the D&D 5e compendium can be filtered and added based on size, name, hit points, and armor class attributes. 
+    <br />
+    <br />
+    This GraphQL API handles the backend functionality including: 
+    <ul align="left">
+      <li><a href="https://www.dnd5eapi.co/docs/#overview--graphql">Consumption of third party D&D 5e GraphQL API</li>
+      <li>Query endpoint(s) for display of created encounters as well as monster details</li>
+      <li>Mutation endpoint(s) for creation of new encounters</li>
+      <li>Current deploy GraphQL API url: https://tavern-keeper-be.onrender.com/</li>
+    </ul>
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be"><strong>Explore Backend docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern-keeper-fe">Frontend Repository</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Request Feature</a>
+  </p>
+</div>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+  1. [About The Project](#about-the-project)
+      - [Built With](#built-with)
+      - [Gems](#gems)
+  2. [Getting Started](#getting-started)
+      - [Installation](#installation)
+      - [Testing](#testing)
+  3. [Database](#database)
+  4. [Endpoints](#endpoints)
+      - [Query: getMonsters](#query-getmonsters)
+      - [Query: getMonster](#query-getmonster)
+      - [Query: getEncounters](#query-getencounters)
+      - [Query: getEncounter](#query-getencounter)
+      - [Mutation: createEncounter](#mutation-createencounter)
+  5. [Roadmap](#roadmap)
+  6. [Contributors](#contributors)
 
-* ...
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+<img src="images/landing.png" alt="landing" width="400" height="200">
+
+[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](INSERT-DEPLOY-LINK)
+
+### Built With
+* [![Ruby on Rails][Rails-shield]][Rails-url]
+* [![GraphQL][GraphQL-shield]][GraphQL-url]
+* [![PostgreSQL][PostgreSQL-shield]][PostgreSQL-url]
+* [![HTML][HTML-shield]][HTML-url]
+* [![JavaScript][JavaScript-shield]][JavaScript-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Gems
+
+#### Deployment
+* [![faraday][gem-faraday]][gem-faraday-url]
+* [![jsonapi-serializer][gem-jsonapi-serializer]][gem-jsonapi-serializer-url]
```

Comment:
I don't think we're using JSONAPI serializer currently. We could probably take this out of the README and remove the gem from our repo.